.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+dist
```

Comment:
- [ ] Kindly remove the dist folder from .gitignore and have it as part of your project structure.